tests/platform_tests/conftest.py

```diff
@@ -41,7 +44,11 @@ def _parse_timestamp(timestamp):
         try:
             time = datetime.strptime(timestamp, FMT_SHORT)
         except ValueError:
-            time = datetime.strptime(timestamp, FMT_ALT)
+            try:
```

Comment:
Considering the different formats we have at this point, maybe something like the following is better:

```python
for format in [FMT, FMT_YEAR, FMT_SHORT, FMT_ALT]:
    try:
        time = datetime.strptime(timestamp, format)
        return time
    except ValueError:
        continue
raise ValueError("Unable to parse {} with any known format".format(timestamp))
```

Reply:
Done in latest.